.pre-commit-config.yaml

```diff
@@ -4,16 +4,15 @@
 files: '\.(py|sh|sql)$'
```

Comment:
Are changes to this file necessary?

Reply:
![image](https://github.com/geopython/pygeoapi/assets/36086324/a662b9f0-a37a-478a-bb4d-91ab3dde8914)

I believe this happened when I synced with the main branch recently. I was not my edit.